### DISCUSSION TITLE
Navigating back after creating a list doesn't clear description in “Create list”

### DISCUSSION THREAD

**Discussion by anderium** [community/community#39731] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If you

1. create a list,
2. save and automatically navigate to it, 
3. then use your history navigation button to go back to the starred repos page
4. create another list

you will create a new list that defaults to use the description of the previously created list.  The name of the list is cleared and shows the grey placeholder.  I would expect the form to clear entirely or even not at all.

Browser is Chrome v107 on MacOS 10.14.6.

**Comment by ossanna16** [comment]
Hi there @anderium and welcome to our community! Thanks for reporting this bug!

**Comment by Tomasgresl17** [comment]
Děkuji

**Comment by Uzocoiny** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> If you
> 
> 1. create a list,
> 2. save and automatically navigate to it, 
> 3. then use your history navigation button to go back to the starred repos page
> 4. create another list
> 
> you will create a new list that defaults to use the description of the previously created list.  The name of the list is cleared and shows the grey placeholder.  I would expect the form to clear entirely or even not at all.
> 
> Browser is Chrome v107 on MacOS 10.14.6.
